examples/wifi-echo/linux/BUILD.gn

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+
+import("${chip_root}/build/chip/tools.gni")
+
+assert(chip_build_tools)
+
+config("includes") {
+  include_dirs = [
+    ".",
+    "include",
+    "${chip_root}/src/app/util",
+  ]
+}
+
+executable("chip-tool-all-clusters-server") {
```

Comment:
Should this be "all-clusters-server" ?

I know we have chip-tool-server but I am also confused by that name.